Dockerfile

```diff
@@ -4,6 +4,9 @@ MAINTAINER Zalando SE
 
 COPY target/essentials.jar /
 
-CMD java $(java-dynamic-memory-opts) $(appdynamics-agent) -jar /essentials.jar
+EXPOSE 8080
+ENV HTTP_PORT=8080
```

Comment:
Probably it's to make sure that Friboo boots Jetty on 8080 and not something else.